lib/Target/X86/X86CallingConv.td

```diff
@@ -854,13 +854,20 @@ def CSR_64_CXX_TLS_Darwin_ViaCopy : CalleeSavedRegs<(sub CSR_64_TLS_Darwin, RBP)
 // All GPRs - except r11
```

Comment:
This is already fixed in a nicer way on the branch, so this is no longer needed.